docs/faststore/components/storefront-features/my-account/configuring-shopper-entity-access.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: "Configuring Shopper entity access"
+hidden: true
+excerpt: "Learn how to configure API credentials to enable Shopper entity data in your B2B FastStore My Account pages."
+---
+
+To enable access to shopper information in My Account pages, such as the data displayed in the **Ordered by** card on the order details page and the shopper filters on the order list page, you need to configure the appropriate permissions and API credentials.
```

Comment:
This filter is going to change, so I think it would be better to remove it from the text, right, @manuellacorreia?

```suggestion
To enable access to shopper information in My Account pages, such as the data displayed in the **Ordered by** card on the order details page, you need to configure the appropriate permissions and API credentials.
```